tools/create-openbsd-vmm-worker.sh

```diff
@@ -52,7 +52,7 @@ cat >etc/installurl <<EOF
 https://${MIRROR}/pub/OpenBSD
 EOF
 
-cat >etc/rc.local <<EOF
+cat >etc/rc.local <<'EOF'
```

Comment:
Since the heredoc is treated as a literal, the escaped `\$` in `hostname \$(cat /etc/myname)` must be dropped. Otherwise it's a syntax error and probably why the subsequent MAKEDEV invocation does not run.

Reply:
Yes, that was exactly the reason. Thanks for noting!